src/hash-table.js

```diff
@@ -7,7 +7,62 @@ class HashTable {
     this.storage = new LimitedArray(this.limit);
     // Do not modify anything inside of the constructor
   }
-  
```

Comment:
Would love to leave more comments in but linter was too strict so I gave up on commenting 😞